### DISCUSSION TITLE
Saml2 Authentication with Samlify

### DISCUSSION THREAD

**Discussion by christopherpickering** [remix-run/remix#5008] [Show and tell / tips]
Howdy, I couldn't find a nice example of how to do saml auth with Remix so put together a demo app using Samlify and Saml-idp to help others get started with SAML and Remix and no third party middle tools.

https://github.com/christopherpickering/remix-saml-authentication

**Comment by humphd** [comment]
@christopherpickering I *literally* arrived here to ask for this exact thing, and here you are, 1 week ago, with working code! There's basically nothing I can find via Google.  I've done this before with Passport.js, but needed a Remix solution.

Amazing, thank you.  Very much appreciated!

**Reply by christopherpickering** [reply]
Thanks! I've made a couple tweaks to make it smoother and will push to that repo. If you use it and can improve, please feel free to send a pr.

**Reply by christopherpickering** [reply]
Really starting to like remix a lot.

**Comment by machour** [comment]
Hey @christopherpickering 
You can contribute your example to our examples repository: https://github.com/remix-run/examples
